tracing/tracer.go

```diff
@@ -102,13 +111,7 @@ func InitGlobalTracer(cfg Config) error {
 
 	tracer.sampleRate = cfg.SampleRate
 	tracer.useHex = cfg.UseHex
-
-	logger := cfg.Logger
-	if logger == nil {
-		logger = log.NopWrapper
-	}
```

Comment:
and then revert this change.

here we decided intentionally to default to `NopWrapper` instead of `DefaultWrapper`